XRTK-Core/Packages/com.xrtk.core/Providers/Controllers/Hands/MixedRealityHandRay.cs

```diff
@@ -0,0 +1,15 @@
+﻿// Copyright (c) XRTK. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using XRTK.Interfaces.InputSystem.Controllers.Hands;
+
+namespace XRTK.Providers.Controllers.Hands
+{
+    /// <summary>
+    /// Default <see cref="IMixedRealityHandRay"/> implementation.
+    /// </summary>
+    public class MixedRealityHandRay : IMixedRealityHandRay
```

Comment:
Still not sure if we need specific hand rays. Let's use the existing `IRayStep` and maybe create a generic ray class for now.